package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-mosaic-component",
-  "version": "3.2.0",
+  "name": "react-mosaic-component-tcn-fork",
```

Comment:
Please revert this and the line below - I'll bump the version as part of the release process (it needs a major version bump for this anyways)